app/Http/Controllers/Service/RegistrationController.php

```diff
@@ -26,9 +33,66 @@ public function index()
     /**
      * Stores an incoming Registration.
      *
-     * @param Request $request
+     * @param StoreNewRegistrationRequest $request
+     * @return \Illuminate\Http\RedirectResponse
      */
-    public function store(Request $request)
+    public function store(StoreNewRegistrationRequest $request)
     {
+        // Duplicate families are fine at this point.
+        $family = new Family(['rvid' => Family::generateRVID()]);
+
+        // Create Carers
+        // TODO: Alter request to pre-join the array?
+        $carers = array_map(
+            function ($carer) {
+                return new Carer(['name' => $carer]);
+            },
+            array_merge(
+                (array)$request->get('carer'),
+                (array)$request->get('carers')
+            )
+        );
+
+        // Create Children
+        $children = array_map(
+            function ($child) {
+                // Note: Carbon uses different time formats than laravel validation
```

Comment:
Is that because you need to set a config var?

Reply:
Apprantly, Carbon uses the formats here.
http://php.net/manual/en/function.date.php

And laravel validation dates use the strtotime formats here.
http://php.net/manual/en/datetime.formats.date.php

Can't see a config option to unify them